src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestRequest.scala

```diff
@@ -1,5 +1,8 @@
 package uk.gov.nationalarchives.tdr.api.utils
 
+import java.sql.PreparedStatement
```

Comment:
There are a few new imports which don't look like they're used?

Reply:
You're right, this class didn't need changing at all, I've put it back to the same as master.